### PR TITLE
Initial recipe for jonga

### PR DESCRIPTION
**Pull request by bwohlberg**
Initial recipe for jonga package passes local build test and appears to be ready to merge. LICENSE file is included with recipe since it was omitted from the MANIFEST in the most recent release available via pip.

### CODE REVIEW

File: recipes/jonga/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  # Skip windows build since pygraphviz dependency not available
+  # noarch: python
```
Comment (jentjr) [inline recipes/jonga/meta.yaml]: Please remove comments
```suggestion

```

File: recipes/jonga/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  # Skip windows build since pygraphviz dependency not available
```
Comment (jentjr) [inline recipes/jonga/meta.yaml]: Please remove comments.
```suggestion

```

File: recipes/jonga/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.3
+    - setuptools
+    - pip
+    - pygraphviz
+    - pytest
```
Comment (jentjr) [inline recipes/jonga/meta.yaml]: Are pytest and pytest-runner needed for the package, or only for tests?

File: recipes/jonga/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+test:
+  imports:
+    - jonga
+
+about:
+  home: https://github.com/bwohlberg/jonga
+  license: GNU General Public v2 or later (GPLv2+)
+  license_family: GPL2
+  license_file: LICENSE
```
Comment (jentjr) [inline recipes/jonga/meta.yaml]: Please refer to the license file in recipe directory using jinja2. https://conda-forge.org/docs/meta.html#packaging-the-license-manually
```suggestion
  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

File: recipes/jonga/meta.yaml (hunk 3 above)
Comment (bwohlberg) [reply recipes/jonga/meta.yaml]: In principle they're only needed for the tests, but the most recent release on PyPi has them listed as install requirements, so they're needed here too, for now.

**Comment by jentjr** [comment]
@conda-forge/staged-recipes I believe this is ready to go. Thanks!

**Comment by bwohlberg** [comment]
Thanks for the review @jentjr.

**Pull request merged**